docarray/array/storage/elastic/getsetdel.py

```diff
@@ -1,9 +1,13 @@
-from typing import Iterable, Dict
+from typing import Iterable, Dict, Sequence
+import math
+import numpy as np
 
 from docarray.array.storage.base.getsetdel import BaseGetSetDelMixin
 from docarray.array.storage.base.helper import Offset2ID
 from docarray import Document
 
+MAX_ES_RETURNED_DOCS = 10000
```

Comment:
lets put this const variable in the `GetSelDelMixin` class as a static attribute

Reply:
noted